services/checkin/models/user_checkin.go

```diff
@@ -1,7 +1,8 @@
 package models
 
 type UserCheckin struct {
-	ID              string                 `json:"id"`
+	UserToken       string                 `json:"userToken" bson:"-" validate:"required"`
+	ID              string                 `json:"id" validate:"omitempty"`
```

Comment:
Same as above

Reply:
Explanation is in previous comment above